src/content/blog/sweet-sprint-01.md

```diff
@@ -0,0 +1,84 @@
+
+![Default behavior of sweet](@assets/blog/sweet-sprint-01/original-behavior.gif)
+
+## Why `sweet`?
+
+Why make this application anyway? Aside from wanting to learn a new language and writing applications for the command line, I wanted to start a touch typing exercise *as quickly as possible* once I got into my terminal. I didn't want to open a web browser to get my reps in.
+
+If you're interested in practicing your typing, I hope you give `sweet` a try, and that it helps you improve!
+
+# Fixes
+
+## Improve argument handling
+
+Previously, I attempted to roll my own solution for handling sub commands and arguments. Writing help messages for each command, and distinguishing between subcommands and flags quickly got out of hand, resulting in incorrect behavior.
+
+To that end, I decided to hand that responsibility to the [Cobra package](https://github.com/spf13/cobra). This simplified my development by allowing me to focus on subcommands and their individual behaviors instead of argument parsing.
+
+# Updates
+
+To that end, I created some new commands, and improved the touch typing game with some new features!
+
+## New commands: `about` and `version`
+
+These commands are useful for displaying information about the program (In truth, these commands were used as a test to understand how Cobra worked). They share some details about `sweet` and its version number, respectively.
+
+## Better exercise inputs
+
+Previously, `sweet` would read files from an exercise folder in a configuration directory, located in `$HOME/.sweet/exercises`. If the file was not located in that directory, then that exercise would not be added in the rotation. Selecting an individual file to practice was also very cumbersome.
+
+I decided to redo how exercises were read into the application, which in turn, made it easier to support different use cases. As of `v0.1.0` of `sweet`, you can do the following things:
+
+### Select a random exercise
+
+This is the same behavior as before, but `sweet` will look for files in `$HOME/.config/sweet/exercises` by default instead of `$HOME/.sweet/exercises`.
+
+![Random exercise](@assets/blog/sweet-sprint-01/new-behavior-default.gif)
+
+### Random exercise from a different directory
+
+If you would like to select a random exercise from a different directory, you can change the `SWEET_EXERCISES_DIR` to a new path!
+
+![Change root directory](@assets/blog/sweet-sprint-01/new-behavior-select-with-env-variable.gif)
+
+### Selecting an exercise from a file
+
+You can now practice any file you'd like with `sweet`! Since some files are pretty large, use the `-s` and `-e` flags to select a start and an end number to practice!
+
+![Select a file from start to end](@assets/blog/sweet-sprint-01/new-behavior-start-and-end-lines.gif)
+
+### Selecting an exercise from STDIN
+
+Pipe output to `sweet` and use the `-` filename to perform an exercise from anywhere you'd like! You can use the `-s` and `-e` flags as well!
```

Comment:
Pipe output of one command into `sweet`, and use the `-` filename to perform an exercise with anything you'd like!